canonicalwebteam/http/__init__.py

```diff
@@ -35,7 +38,7 @@
     LATENCY_HISTOGRAM = None
 
 
-class TimeoutHTTPAdapter(requests.adapters.HTTPAdapter):
+class TimeoutHTTPAdapter(adapters.HTTPAdapter):
```

Comment:
Have we tested that the timeout still works? It needs to

Reply:
Nope. I have noted it down. But wanted to wait until our meeting to discuss the best approach.
My idea was to create a new sdapter that mixes the **CacheControl** and **TimeOut** adapters.